frontend/src/lib/components/icons.tsx

```diff
@@ -679,8 +679,8 @@ interface InsightIconProps {
 }
 function InsightIcon({ background = '#747EA2', noBackground = false, children }: InsightIconProps): JSX.Element {
     return (
-        <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
-            {!noBackground ? <rect width="32" height="32" rx="4" fill={background} /> : null}
+        <svg width="1em" height="1em" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Now all insight icons can change size depending on the `font-size`

Reply:
Do we really need this? Our icons should be pretty standardized to a 24px x 24px viewbox with standard sizing of the paths within. My concern is that it leads to inconsistent icon sizes.

Reply:
Well we had them at 32px (which we use for the New Insight dropdown) and at least in this case we require a smaller size that matches font size. I think it's fine to do this as we do it for every other icon we have. One thing we could do to avoid inconsistencies is have a discrete set of sizes (e.g. `sm`, `md`, `lg`) instead of allowing the continuous spectrum of any size. Thoughts?